RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -725,6 +725,7 @@ boolean doBedtime()
 	}
 
 	//We are committing to end of day now...
+	auto_scepterRollover();
```

Comment:
should be after line 1314, both to avoid summoning a baywatch when you have better equip (need to add checks for this), and to only do off-hander's day when your desired rollover offhand is +adv